Apigee/SmartDocs/Revision.php

```diff
@@ -343,7 +351,7 @@ public function load($revisionId = null)
         if (is_int($revisionId) && $revisionId < 1) {
             throw new ParameterException('Cannot load a revision number less than 1.');
         }
-        $this->get($revisionId . '?expand=true');
+        $this->get(rawurlencode($this->apiId) . '/revisions/' . $revisionId . '?expand=true');
```

Comment:
Notice how all of these calls had to be modified to put /revisions back in, except your two new methods.  Move them to the correct object instead.